modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -5,20 +5,27 @@
 [id="installation-osp-kuryr-known-limitations_{context}"]
 = Known limitations of installing with Kuryr
 
-There are known limitations when using Kuryr SDN:
+There are known limitations when using Kuryr SDN, depending on the
+{rh-openstack} version:
 
 * An Amphora load balancer VM is deployed per OpenShift Service with
-the default Octavia load balancer driver (Amphora driver). If the environment is
-resource constrained, creating a large amount of Services could be a problem.
-* Depending on the Octavia version, UDP listeners are not supported. This means
-that OpenShift UDP Services are not supported.
-* There is a known limitation of Octavia not supporting listeners on different
-protocols, like UDP and TCP, on the same port. Thus, Services exposing the same
-port for different protocols are not supported.
+the default Octavia load balancer driver (Amphora driver). In a
+resource-constrained environment, creating too many Services can cause
+problems.
+* Octavia {rh-openstack} version 13 does not support UDP listeners. This means
+that OpenShift UDP services are not supported. UDP listeners a re supported
+from Octavia {rh-openstack} version 16 and later.
+* Octavia {rh-openstack} version 13 cannot listen to multiple protocols,
+like UDP and TCP, on the same port. Service that expose the same port to
+different services are not supported. Listeners that use different protocols
+are supported in {rh-openstack} version 16 and later. Note this only applies
```

Comment:
Suggestion:

[NOTE]
Only the Octavia Amphora driver supports different protocols on the same port in {rh-openstack} version 16. The OVN Octavia driver does not support listeners that use different protocols.